tests/framework/installer/ceph_installer.go

```diff
@@ -129,6 +139,13 @@ func (h *CephInstaller) CreateRookOperatorViaHelm(namespace, chartSettings strin
 
 	}
 
+	if err = h.WipeClusterDisks(namespace); err != nil {
```

Comment:
The helm chart starts the operator on line 136 above, so this would need to be done first. But if you don't need the discovery results in the helm test, it's not critical since it would be more complicated to create the namespace too.

Reply:
This could also call the new helper method